test/integration-test.sh

```diff
@@ -298,3 +298,65 @@ for i in `seq 1 10`; do
   log "Node 1 send message to node 5"
   send_message "${api1}" "${addr5}" "hello, world" "" 600
 done
+
+# redeem tickets
+log "Node 2 should redeem all tickets"
+result=$(run_command "${api2}" "redeemTickets" "Redeemed all tickets" 600)
+log "--${result}"
+
+log "Node 3 should redeem all tickets"
+result=$(run_command "${api3}" "redeemTickets" "Redeemed all tickets" 600)
+log "--${result}"
+
+log "Node 4 should redeem all tickets"
+result=$(run_command "${api4}" "redeemTickets" "Redeemed all tickets" 600)
+log "--${result}"
+
+log "Node 5 should redeem all tickets"
+result=$(run_command "${api5}" "redeemTickets" "Redeemed all tickets" 600)
+log "--${result}"
+
+# initiate channel closures
+log "Node 1 close channel to Node 2"
+result=$(run_command "${api1}" "close ${addr2}" "Initiated channel closure" 600)
+log "-- ${result}"
+
+log "Node 2 close channel to Node 3"
+result=$(run_command "${api2}" "close ${addr3}" "Initiated channel closure" 600)
+log "-- ${result}"
+
+log "Node 3 close channel to Node 4"
+result=$(run_command "${api3}" "close ${addr4}" "Initiated channel closure" 600)
+log "-- ${result}"
+
+log "Node 4 close channel to Node 5"
+result=$(run_command "${api4}" "close ${addr5}" "Initiated channel closure" 600)
+log "-- ${result}"
+
+log "Node 5 close channel to Node 1"
+result=$(run_command "${api5}" "close ${addr1}" "Initiated channel closure" 600)
+log "-- ${result}"
+
+# close channels
+echo "Waiting 1 minure for cool-off period"
```

Comment:
Could we safe ourselves this additional static wait period by waiting for a logs entry to appear and then run the verification steps you have already below?

Reply:
Logs entry from where?

Reply:
> Logs entry from where?

From each of the node log files. We have a helper function which you can use to grep and wait for log entries in there.

Reply:
Yeah but which log? AFAIK there is no log indicating the end of the log period

Reply:
ok, let's keep it as is for now